test/unit/helpers/dom/Form.js

```diff
@@ -6,6 +6,10 @@ define(['okta/jquery', 'okta/underscore', './Dom'], function ($, _, Dom) {
       return this.el('o-form-head').trimmedText();
     },
 
+    errorBannerText: function () {
```

Comment:
can you use any of the error functions we already have? I think ```errorMessage``` should work, but we also have ```hasErrors```, ```errorBox``` and ```error```

Reply:
Makes sense. I'll update the test